clifford/__init__.py

```diff
@@ -232,43 +232,6 @@ def grade_obj_func(objin_val, gradeList, threshold):
     return np.argmax(modal_value_count)
 
 
-def general_exp(x, max_order=15):
```

Comment:
Strictly you now need to do `from .taylor_expansions import general_exp` to make this still publicly accessible.

Reply:
Or ``from .taylor_expansions import exp as general_exp` with my suggestion below. Or even:
```python
def general_exp(x, **kwargs):
   warnings.warn("cf.general_exp is deprecated. Use `mv.exp()` or `np.exp(mv)` on multivectors, or `cf.taylor_expansions.exp(x)` on arbitrary objects", DeprecationWarning)
   return taylor_expansions.exp(x, **kwargs)
```